scene/3d/gpu_particles_3d.cpp

```diff
@@ -499,6 +500,10 @@ GPUParticles3D::TransformAlign GPUParticles3D::get_transform_align() const {
 	return transform_align;
 }
 
+void GPUParticles3D::update_layer_mask() {
```

Comment:
Perhaps this should take ``uint32_t p_mask`` as an argument. Right now it seems kind of weird to expose this as a function as it doesn't do anything that the user can control.

Reply:
I added p_mask as an argument as you said. Thanks for the feedback.